app/components/widgets/forms/ticket-input.js

```diff
@@ -6,6 +6,11 @@ export default Component.extend({
   actions: {
     toggleSettings() {
       this.toggleProperty('isExpanded');
+    },
+    minMaxCheck(ticket) {
+      if (ticket.minPrice > ticket.maxPrice) {
+        this.notify.error(this.l10n.t('Minimum price should be less than maximum price'));
```

Comment:
Nope. No notification. Instead error message should be shown with the element itself

Reply:
<img width="726" alt="Screenshot 2019-07-04 at 7 39 32 AM" src="https://user-images.githubusercontent.com/25197147/60635071-f5246c80-9e2e-11e9-8af9-61aa623f9ae5.png">
I've used the semantic tooltip for this